fix: Show tooltip on defined widget (#3071)
kyma-project/busola: docs/extensibility/60-form-widgets.md

| **enum[]**            | No       | an array of options | Generate an input field with a dropdown. Optionally can be a string containing a [JSONata](jsonata.md) expression returning an array of options.                                                                             |
| **placeholder**       | No       | string              | Specifies a short hint about the input field value.                                                                                                                                                                          |
| **required**          | No       | boolean             | Specifies if a field is required. The default value is taken from CustomResourceDefintion (CRD); if it doesn't exist in the CRD, then it defaults to `false`.                                                                |
| **inputInfo**         | No       | string              | A string below the input field that shows how to fill in the input. You can use the {{ [`name`] (`link`) }} format to display a `name` instead of a `link.                                                                   |

> | **description**       | No       | string              | A string displayed in a tooltip when you hover over a question mark icon, next to the input's label. The default value is taken from the CustomResourceDefintion (CRD). The tooltip can be disabled by placing empty string. |

Reviewer: ```suggestion
| **description**       | No       | string              | A string displayed in a tooltip when you hover over a question mark icon, next to the input's label. The default value is taken from the CustomResourceDefintion (CRD). To disable the tooltip, add an empty string. |
```
